cmd/api/src/api/middleware/etac.go

```diff
@@ -22,7 +22,7 @@ import (
 
 	"github.com/gorilla/mux"
 	"github.com/specterops/bloodhound/cmd/api/src/api"
-	"github.com/specterops/bloodhound/cmd/api/src/api/v2"
+	v2 "github.com/specterops/bloodhound/cmd/api/src/api/v2"
```

Comment:
pfc

Reply:
Does the 'v2' prefix need to be removed?

Reply:
Unfortunately required